src/Components/WebAssembly/WebAssembly.Authentication/test/RemoteAuthenticationServiceTests.cs

```diff
@@ -596,6 +596,6 @@ internal class TestNavigationManager : NavigationManager
         public TestNavigationManager() =>
             Initialize("https://www.example.com/base/", "https://www.example.com/base/add-product");
 
-        protected override void NavigateToCore(string uri, bool forceLoad) => throw new NotImplementedException();
+        protected override void NavigateToCore(string uri, NavigationOptions options) => throw new NotImplementedException();
```

Comment:
Looks like the change here and the next few files can all be reverted with the above suggestions.